util/update-gh-pages-docs.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Do not run this script on forked repos, not on pull requests, only on master
+if [ "$TRAVIS_REPO_SLUG" == "RADAR-CNS/ManagementPortal" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_BRANCH" == "master" ]; then
```

Comment:
Perhaps, instead of this line, you can run this script as a Travis deployment target with `script` provider, only on the master branch.